packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -149,6 +149,19 @@ export interface QueueProcessingServiceBaseProps {
    */
   readonly scalingSteps?: ScalingInterval[];
 
+  /**
+   * Grace period after scaling activity in seconds.
+   *
+   * Subsequent scale outs during the cooldown period are squashed so that only
+   * the biggest scale out happens.
+   *
+   * Subsequent scale ins during the cooldown period are ignored.
+   *
+   * @see https://docs.aws.amazon.com/autoscaling/application/APIReference/API_StepScalingPolicyConfiguration.html
+   * @default No cooldown
```

Comment:
```suggestion
   * @default 300 seconds
```

https://docs.aws.amazon.com/autoscaling/application/APIReference/API_StepScalingPolicyConfiguration.html#API_StepScalingPolicyConfiguration_Contents